doc/man3/flux_job_timeleft.rst

```diff
@@ -0,0 +1,42 @@
+====================
+flux_job_timeleft(3)
+====================
+
+
+SYNOPSIS
+========
+
+::
+
+   #include <flux/core.h>
+
+::
+
+   int flux_job_timeleft (flux_t *h,
+                          flux_error_t *error,
+                          double *timeleft);
```

Comment:
I see a comment below that says *timeleft is in units of seconds, but I don't see that specified in this man page.

Reply:
Great catch! Thanks, will fix.